### PR TITLE
Fixed the height of card when 1 bookmark selected

### LINKED ISSUE #2661: (SP: 0.5) Fix card height
Fix the height of card when only 1 bookmark selected
[Figma](https://www.figma.com/design/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?node-id=20999-101212&t=qnNKyZyMuTiTvkwl-4)

image

### PR DESCRIPTION
**Pull request by VKormylo**
Fixed the height of card when 1 bookmark selected
Now it looks like this
image

### CODE REVIEW

**Review by ipasic-softserve — COMMENTED** [review]
When on grid view, the gap is still present:

image

**Comment by VKormylo** [comment]
Is it okay like this now?
Gap in the first one is slightly bigger because it has less lines in title than others
image

**Comment by Renatavl** [comment]
image

@VKormylo PR is related to fixing the height of card but there are a lot of files added which do not relate to this issue

**Comment by VKormylo** [comment]
Other person's commit accidentally got added to the branch, I'm going to fix that.

**Review by ipasic-softserve — APPROVED** [review]

File: src/containers/find-offer/offer-container/OfferContainer.styles.ts [hunk 1]
```diff
@@ -12,9 +12,13 @@ export const styles = {
...
   gridItem: { width: '100%' },
-  appCard: { padding: { sm: '30px 20px', md: '30px 20px' } },
+  appCard: {
+    padding: { sm: '30px 20px', md: '30px 20px' },
+    alignSelf: 'flex-start'
+  },
   appCardSquare: {
     minHeight: '460px',
-    padding: '24px 20px'
+    padding: '24px 20px',
```
Comment (Mav-Ivan) [inline src/containers/find-offer/offer-container/OfferContainer.styles.ts]: better to use `p:` instead of `padding`

**Review by Mav-Ivan — APPROVED** [review]

**Review by Renatavl — APPROVED** [review]

**Pull request merged**